docs/visual-basic/programming-guide/concepts/linq/how-to-split-a-file-into-many-files-by-using-groups-linq.md

```diff
@@ -62,7 +62,7 @@ Class SplitWithGroups
         ' Note that nested foreach loops are required to access  
         ' individual items with each group.  
         For Each gGroup In groupQuery  
-            Dim fileName As String = "..'..'..'testFile_" & gGroup.groupKey & ".txt"  
+            Dim fileName As String = "../../../testFile_" & gGroup.groupKey & ".txt"  
```

Comment:
In Visual Studio, this writes files to the parent directory of the project directory. That's also true of lines 48-49: they attempt to read files from the parent of the project directory.